content/blog/2019/02/2019-02-05-jenkins-new-year-in-china.adoc

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: "Jenkins new year in China"
+tags:
+- core
+- community
+author: linuxsuren
+---
+
+At the time of the Spring Festival. I want to make a summary of some activities in the last year. 
+You might already notice that more and more Chinese contributors emerge in the Jenkins community. 
+We have a GSoC champion who is link:https://github.com/cizezsy[Shenyu Zheng]. 
+He is a great example for other students. With the effort of three skilled engineers, 
+many Jenkins users could learn the edge technologies and useful use cases. 
+They co-orgnaized several Jenkins Meetups in a couple of cities in China.
+
+There are two workshops about Jenkins and Jenkins X in the DevOps International Summit. link:https://github.com/rawlingsj[James Rawlings] gave us a wonderful view of the Jenkins X. Many people start to know this project. The link:https://jenkins-x.io/zh/[Chinese website of jx] would be helpful to those people.
+
+On November 3rd, 2018 the Jenkins User Conference China(JUCC) was hosted in Shenzhen. More than 200 attendees gathered at JUCC to share and discuss Jenkins, DevOps, Continuous Delivery, Pipeline, and Agile.
+
+There was a Jenkins workshop to teach users to develop a plugin in October. It was during the Hacktoberfest 2018. So some people got a beautiful T-shirt at this meetup. We'll keep this event in 2019. I hope more users and developers could join us.
+
+Thank you all folks. And other friendly contributors.
+
+Chinese is our main communication language. A large number of the Jenkins users are not a proficient English speaker. 
+So letting most of Chinese Jenkins users could easily use Jenkins as their CI/CD platform is the final mission of link:https://jenkins.io/sigs/chinese-localization/[Chinese Localization SIG]. 
+You can find three participants on the page. But that's not the link:https://github.com/jenkins-infra/cn.jenkins.io/graphs/contributors[full list]. 
+More exciting thing is that link:https://github.com/alauda[Alauda] giving a big support wich as a startup company.
```

Comment:
This all looks great, thanks! Minor typo on this line, maybe remove word 'wich'. Otherwise, lgtm.